test/clj_puppetdb/http_test.clj

```diff
@@ -52,23 +52,57 @@
         response-data ["node-1" "node-2"]
         response-data-encoded (json/encode response-data)
         response-headers {"x-records" (.toString (count response-data))}
-        fake-get (fn[status] {:status status :body (io/input-stream (.getBytes response-data-encoded)) :headers response-headers})]
+        fake-get (fn [status] {:status status :body (io/input-stream (.getBytes response-data-encoded)) :headers response-headers})]
 
-      (testing "Should have proper response"
-        (with-redefs [http/request-with-client (fn[_ _ _] (future (fake-get 200)))]
-          (let [GET-response (GET client path params)]
-            (is (= (first GET-response) response-data))
-            (is (= (second GET-response) response-headers)))))
+    (testing "Should have proper response"
+      (with-redefs [http/request-with-client (fn [_ _ _] (future (fake-get 200)))]
+        (let [GET-response (GET client path params)]
+          (is (= (first GET-response) response-data))
+          (is (= (second GET-response) response-headers)))))
 
-      (testing "Should throw proper exception"
-        (with-redefs [http/request-with-client (fn[_ _ _] (future (fake-get 400)))]
-          (try
-            (GET client path params)
-            (catch ExceptionInfo ei
-              (let [info (.getData ei)]
-                (is (= (:status info) 400))
-                (is (= (:kind info) :puppetdb-query-error))
-                (is (= (:params info) params))
-                (is (= (:endpoint info) path))
-                (is (= (:host info) host))
-                (is (= (:msg info) response-data-encoded)))))))))
+    (testing "Should throw proper exception"
+      (with-redefs [http/request-with-client (fn [_ _ _] (future (fake-get 400)))]
+        (try
+          (GET client path params)
+          (catch ExceptionInfo ei
+            (let [info (.getData ei)]
+              (is (= (:status info) 400))
+              (is (= (:kind info) :puppetdb-query-error))
+              (is (= (:params info) params))
+              (is (= (:endpoint info) path))
+              (is (= (:host info) host))
+              (is (= (:msg info) response-data-encoded)))))))))
```

Comment:
Agree. The problem is that I used a default formatter in IntelliJ/Cursive to format my code but it formatted even some other code which wasn't properly formatted. That could be a bit difficult to revert. :-(

However, there is a handy parameter in github. If you use "?w=1" all the whitespace changes are filtered out. So, in  this case https://github.com/puppetlabs/clj-puppetdb/pull/22/files?w=1